src/docs/sdk/event-payloads/contexts.mdx

```diff
@@ -693,6 +693,23 @@ Additional information that allows Sentry to connect multiple transactions, span
 
 : _Optional_. The origin of the trace indicates what created the trace. For more details, see <Link to="/sdk/performance/trace-origin">trace origin</Link>.
 
+`data`
+
+: _Optional_. The map of custom provided data. Currently Sentry SDKs can set following fields:
+
+| Field            | Type              | Description                                                  |
+| ---------------- | ----------------- | ------------------------------------------------------------ |
+| `route`          | `string` or `map` | The current route in the application.                        |
+| `previous_route` | `string` or `map` | The previous route in the application the user was visiting. |
+
+The `route` can also have currently following predefined fields if it is a map:
```

Comment:
```suggestion
The `route` currently supports the following predefined fields:
```

I had to re-read this sentence when I first read it, but tbh I don't know if my suggestion is an upgrade...